### PR TITLE
add command-line options

### LINKED ISSUE #8: Optional Parameters
I really love your component! I experienced sub second latencies even using WiFi.
I think it would be nice if one could use WiFi as an automatic fallback (if the ip is provided in the script) if usb is not available.
And also a command line option to specify if you want to use horizontal or vertical mode. 
In my opinion these two things would be quite handy. What do you think?

I could implement this and open a PR, if you want me to.

Thanks again for this awesome method!

### PR DESCRIPTION
**Pull request by AaronDavidSchneider**
Remove fallback wifi and use more sophisticated commandline options

Closes #10 
Closes #8

### CODE REVIEW

File: reStream.sh [hunk 1]
```diff
@@ -3,10 +3,23 @@
...
+
+# Loop through arguments and process them
+for arg in "$@"
+do
+    case $arg in
+        -p|--portrait)
+        landscape=false
+        shift
+        ;;
+        -h|--host)
```
Comment (AaronDavidSchneider) [inline reStream.sh:15]: -h might be a bit irritating

Comment (rien) [reply reStream.sh:15]: Hmm yes, it could cause confusion with `--help`. Do you have suggestions for another abbreviation?

Comment (AaronDavidSchneider) [reply reStream.sh:15]: maybe `-s` for `--ssh`, or `-r` for `--remote`

Comment (rien) [reply reStream.sh:15]: Maybe `-d|--destination`?

Comment (rien) [reply reStream.sh:15]: You can choose :)

**Comment by AaronDavidSchneider** [comment]
You will need to adapt the readme. I didnt do this in this PR yet.

**Pull request merged**